src/services/OrderService.ts

```diff
@@ -290,6 +290,38 @@ const retryShippingLabel = async (shipmentIds: Array<string>): Promise<any> => {
   })
 }
 
+const fetchShipmentLabelError = async (shipmentIds: Array<string>): Promise<any> => {
+  let shipmentLabelError = [];
+  const params = {
+    "entityName": "ShipmentPackageRouteSeg",
+    "inputFields": {
+      "shipmentId": shipmentIds,
+      "shipmentId_op": "in",
+      "gatewayMessage": null,
+      "gatewayMessage_op": "notEqual",
+      "gatewayStatus": "error", 
+      "gatewayStatus_op": "equals"
+    }
```

Comment:
Add a viewSize of 20. Also pass fieldList to only fetch the required information from the entity, in our case just fetch shipmentId and gatewayMessage